GraknConsole.java

```diff
@@ -261,19 +254,31 @@ private void runQuery(GraknClient.Transaction tx, String queryString) {
             terminal.handle(Terminal.Signal.INT, s -> answerPrintingJob.cancel(true));
             answerPrintingJob.get();
             Instant end = Instant.now();
-            printer.info("answers: " + counter + ", duration: " + Duration.between(start, end).toMillis() +" ms");
-        } catch (InterruptedException | ExecutionException e) {
+            printer.info("answers: " + counter + ", duration: " + Duration.between(start, end).toMillis() + " ms");
+        } catch (InterruptedException e) {
             e.printStackTrace();
+        } catch (ExecutionException e) {
+            throw (RuntimeException)e.getCause();
```

Comment:
@lolski should it be ClientJavaException maybe?

Reply:
I think we should just throw it as is.

Why do we need to cast it to `RuntimeException` in the first place?

Reply:
@lolski Otherwise I need to add `throws Throwable` in all parent functions. I think it should be ok to cast it to `RuntimeException`, since the only exception we expect is `GraknClientException`. If there's a different exception, then there's a bug in the client.

So maybe we can also cast it to `GraknClientException`.

Reply:
I think let's do this instead which I think is nicer since we do zero assumptions regarding what type may be thrown:

```
catch (ExecutionException e) {
  throw new RuntimeException(e);
}
```

Reply:
@lolski We cannot do that because we need to throw an exception of type `GraknCientException` for the parent function to know it can print error and close transaction REPL.

Reply:
I see, got it. let's throw `GraknClientException`.